src/main/java/seedu/address/storage/JsonAddressBookStorage.java

```diff
@@ -45,17 +51,36 @@ public Optional<ReadOnlyAddressBook> readAddressBook() throws DataLoadingExcepti
     public Optional<ReadOnlyAddressBook> readAddressBook(Path filePath) throws DataLoadingException {
         requireNonNull(filePath);
 
-        Optional<JsonSerializableAddressBook> jsonAddressBook = JsonUtil.readJsonFile(
-                filePath, JsonSerializableAddressBook.class);
-        if (!jsonAddressBook.isPresent()) {
-            return Optional.empty();
-        }
-
         try {
```

Comment:
Code could be made a little more streamlined by using a single try catch clause instead of nested try catches, especially when the inner catch have the same Exception type and message. However, I also understand wanting to maintain the flow of logic of code, so it is not necessary to change this :)